tests/scripts/thread-cert/thread_cert.py

```diff
@@ -483,14 +483,19 @@ def _parse_params(self, params: Optional[dict]) -> dict:
 
         if params.get('is_bbr') or params.get('is_otbr'):
             # BBRs must use thread version 1.2
-            assert params.get('version', '1.2') == '1.2', params
-            params['version'] = '1.2'
+            version = params.get('version', '1.2')
```

Comment:
```suggestion
            version = params.get('version', '1.3')
```
Should be 1.3 by default? 

or maybe we can define a const like `consts.LATEST_THREAD_VERSION` ?